### ISSUE TITLE
Code does not handle Saturday and Sunday

### ISSUE THREAD

**Issue by neilpeterson** [neilpeterson/redmond-dining-bot#20]
Have put a temp workaround in place, rework this as priority allows.

``` none
// temporary dev fix for weekend
dayofweek.Add(6, "FRI");
dayofweek.Add(7, "FRI");
```

**Comment by jongeller** [comment]
Where is this issue manifesting? Is it something in LUIS, or is it something we can work on post intent return from the LUIS api?

**Comment by neilpeterson** [comment]
Not LUIS related, should be an easy fix.

We currently return the day of week, and then use this is a refdining API call. The API will not take SAT or SUN, hence I discovered the issue this weekend. Simple detection of it being a weekend and sending a reply of "MSFT dining is closed on the Weekend" or similar should work.

``` none
DateTime day = DateTime.Now;
int today = (int)day.DayOfWeek;

httpClient.GetAsync("https://msrefdiningint.azurewebsites.net/api/v1/menus/building/" + newid + "/weekday/" + today);
```

**Comment by jongeller** [comment]
cool, on it now

**Comment by jongeller** [comment]
resolved in 157d2ba waiting approval for pull request.
